doc/source/rllib/index.rst

```diff
@@ -39,59 +41,50 @@ RLlib in 60 seconds
 
 .. figure:: images/rllib-index-header.svg
 
-
-It only takes a few steps to get your first RLlib workload up and running on your laptop:
-
-**TensorFlow or PyTorch**:
+It only takes a few steps to get your first RLlib workload
+up and running on your laptop.
 
 RLlib does not automatically install a deep-learning framework, but supports
-TensorFlow (both 1.x with static-graph and 2.x with eager mode) as well as
-PyTorch. Depending on your needs, make sure to install either TensorFlow or
-PyTorch (or both as shown below):
-
-.. code-block:: bash
-
-    $ conda create -n rllib python=3.8
-    $ conda activate rllib
-    $ pip install "ray[rllib]" tensorflow torch
-
-Note, for installation on computers running Apple Silicon (such as M1), please follow instructions
-`here <https://docs.ray.io/en/latest/installation.html#m1-mac-apple-silicon-support>`_
-
-To be able to run our Atari examples, you should also install:
-
-.. code-block:: bash
-
-    $ pip install "gym[atari]" "gym[accept-rom-license]" atari_py
-
-After these quick pip installs, you can start coding against RLlib.
-
-Here is an example of running a PPO Trainer on the `Taxi domain <https://www.gymlibrary.dev/environments/toy_text/taxi/>`_
-for a few training iterations, then perform a single evaluation loop
-(with rendering enabled):
-
-.. literalinclude:: ../../../rllib/examples/documentation/rllib_in_60s.py
+**TensorFlow** (both 1.x with static-graph and 2.x with eager mode) as well as
+**PyTorch**.
+Depending on your needs, make sure to install either TensorFlow or
+PyTorch (or both, as shown below):
+
+.. raw:: html
+
+    <div class="termynal" data-termynal>
+        <span data-ty="input">pip install "ray[rllib]" tensorflow torch</span>
+    </div>
+
+.. margin::
+
+    For installation on computers running Apple Silicon (such as M1), please follow instructions
+    `here <https://docs.ray.io/en/latest/installation.html#m1-mac-apple-silicon-support>`._
+    To be able to run our Atari examples, you should also install
+    `pip install "gym[atari]" "gym[accept-rom-license]" atari_py`.
+
+This is all you need to start coding against RLlib.
+Here is an example of running a PPO Algorithm on the
+`Taxi domain <https://www.gymlibrary.dev/environments/toy_text/taxi/>`_.
+We first create a `config` for the algorithm, which sets the right environment, and
+defines all training parameters we want.
+Next, we `build` the algorithm and `train` it for a total of `5` iterations.
+A training iteration includes parallel sample collection by the environment workers, as well as loss calculation on the collected batch and a model update.
+As a last step, we `evaluate` the trained Algorithm and and render each timestep to the shell's output:
```

Comment:
`and and` <- remove one `and`

Reply:
Actually, we don't render anymore due to some gym-problems with newer gym versions.

Reply:
ok, fixed